turltesim_cleaner/src/robot_cleaner.cpp

```diff
@@ -0,0 +1,129 @@
+#include "ros/ros.h" //library with ros services
+#include "geometry_msgs/Twist.h" // contains the ros message type Twist
+#include "turtlesim/Pose.h" // this allows the node read the pose message type published by the node
```

Comment:
looks good as an initial prototype. This could benefit from creating a "spiral search" class so it's modular. That way we can transfer it from the sim directly to the robot.